homework/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+        <meta charset="UTF-8">
+        <title>HUDA ALISMAIL</title>
+        <link rel="stylesheet" href ="styles/styles.css">
+
+    <body>
+
+        <div class="name-section">          
+              <h1> Joe Johnson</h1>
+              <span class="border"></span>
+        
+
+        <div class="pro-photo">
+        <img src ="styles/profi.jpg">
+        </div>
+
+      
+        <header class="pages">
+        
+            <a href ="index.html">About Page</a>
```

Comment:
use file names that are easier to differentiate for example 
portfolio.html, aboutMe.html